nix/nodes/common/services/cvat-README.md

```
@@ -0,0 +1,215 @@
+Para verificar o status de todos os serviços:
+
+```bash
+sudo systemctl status cvat.service
+sudo systemctl status podman-cvat-*
+```
+
+### Acessando a Interface
+
+O CVAT estará disponível via ts-proxy no endereço:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)